### ISSUE TITLE
[Question] Error in getModelJSON with React

### ISSUE THREAD

**Issue by ajaykrupalk** [xenova/transformers.js#364]
Hey, I am trying to transcribe audio to speech using transformers.js. I tried two ways 

1.  https://huggingface.co/docs/transformers.js/api/pipelines#pipelinesautomaticspeechrecognitionpipeline
2. https://huggingface.co/docs/transformers.js/tutorials/react

But seem to get an error like this

![image](https://github.com/xenova/transformers.js/assets/67155124/bfa37f1b-6b57-42f9-8792-8542fc2fc958)

Files for your reference: https://filebin.net/88munmsfk4u0127m

Please do let me know if I am doing something wrong or what is the best way using ReactJS

**Comment by xenova** [comment]
Could you share the code you are using to load the model? Also, are you trying to access the model files from the HF hub, or rather, are they locally hosted?

**Comment by ajaykrupalk** [comment]
@xenova Is there any update on this? Awaiting your response.

**Comment by maudnals** [comment]
Hey folks, I'm experiencing a similar issue. 
My setup isn't React-based though so I've filed it separately for clarity, in case the root cause is actually different:  https://github.com/xenova/transformers.js/issues/366

**Comment by xenova** [comment]
@ajaykrupalk Could you try what I suggested [here](https://github.com/xenova/transformers.js/issues/366#issuecomment-1774037601)?

TLDR: Skip the local file check and add
```js
import { pipeline, env } from '@xenova/transformers';
env.allowLocalModels = false;
```

**Comment by ajaykrupalk** [comment]
Hi @xenova I think the issue gets fixed if I start using vite + react. I guess I should migrate my app to vite now 😢😅
